.meta/_partials/fields/_kafka_options.toml.erb

```diff
@@ -26,3 +26,39 @@ examples = [
 description = """\
 The options and their values. Accepts `string` values.
 """
+
+[<%= namespace %>.sasl]
+type = "table"
+category = "SASL"
+common = false
+description = "Options for SASL/SCRAM authentication support."
+
+[<%= namespace %>.sasl.children.enabled]
```

Comment:
Do you think we can more intelligently detect if it needs to enabled? Like if username/password is present?

Reply:
Yes, I think better detect sasl by `username`/`password`/`mechanism`. But I added `enabled` here for consistency with `tls` option.